apps/src/templates/StyleGuide.jsx

```diff
@@ -59,11 +59,11 @@ function Colors() {
       </p>
       <p>
         JavaScript usage is as follows:
-        <pre>
-          {`import colors from '@cdo/apps/colors';
-console.log("the hex value for purple is:", colors.purple);`}
-        </pre>
       </p>
+      <pre>
+        {`import colors from '@cdo/apps/colors';
+console.log("the hex value for purple is:", colors.purple);`}
```

Comment:
nope. This shows up in the style guide as example code. It doesn't actually run.